AppFramework/Core/GREYElementInteraction.m

```diff
@@ -193,7 +194,7 @@ - (void)matchElementsWithTimeout:(CFTimeInterval)timeout
   } else if (searchActionError) {
     NSString *searchActionDescription = searchActionError.localizedDescription;
     NSString *description =
-        [NSString stringWithFormat:@"Search action failed: %@\n", searchActionDescription];
```

Comment:
Was this an accidental change?

Reply:
No, this is intentional. This newline character was interfering with the way ErrorFormatter inserts a new line character between each key.